### PR TITLE
Crop to top 30 apps for bar charts.

### LINKED ISSUE #43: Fix displaying of time active/visible/open for many applications
# Description

When using a lot of applications throughout the day, the resulting time graphs can’t be displayed correctly in the given 200px of height:

timegraph

# Expectation

I would have expected one of the following:
* rows that don’t fit get cut off, because they have too little time on them
* rows that don’t fit get accumulated as a single row as “Other”
* the graph has enough height to fit all rows

For me personally, I would be fine with any variation where I can still read the entries.

# Reproducing

Use the default way to track thyme and do a lot of context switching thoughout your work day:
```
watch -n 30 thyme track -o thyme.json
thyme show -i thyme.json -w stats > thyme.html
```

### PR DESCRIPTION
**Pull request by paulirish**
Fixes #43 

Screenshot (zoomed out):

image

### CODE REVIEW

File: show.go [hunk 1]
```diff
@@ -35,9 +35,9 @@ type AggTime struct {
 
 // NewAggTime returns a new AggTime created from a Stream.
 func NewAggTime(stream *Stream, labelFunc func(*Window) string) *AggTime {
-	active := NewBarChart("Active", "App", "Samples", "Time (multiplied by number of windows) application was active")
-	visible := NewBarChart("Visible", "App", "Samples", "Time (multiplied by number of windows) application was visible")
-	all := NewBarChart("All", "App", "Samples", "Time (multiplied by number of windows) application was open")
+	active := NewBarChart("Active", "App", "Samples", "Top 10 active applications by time (multiplied by window count)")
```
Comment (raboof) [inline show.go]: I guess it's time to put this number in a variable :)

Comment (paulirish) [inline show.go]: good catch :)

Comment (paulirish) [inline show.go]: Welp, updated the strings. 

I suck at go too much to do this the right way. :) If you wanna take a crack, I can merge in your commit or whatever

**Comment by paulirish** [comment]
And a commit from @raboof makes this PR a combo effort. :D

File: show.go [hunk 2]
```diff
@@ -4,10 +4,13 @@ import (
 	"fmt"
 	"os"
 	"sort"
+	"strconv"
 	"text/template"
 	"time"
 )
 
+var numberOfBars = 30
```
Comment (beyang) [inline show.go]: `const numberOfBars = 30`

**Comment by beyang** [comment]
Thanks for the submission!

**Pull request merged**